charts/incubator/rollarr/Chart.yaml

```diff
@@ -25,5 +21,11 @@ sources:
   - https://github.com/truecharts/charts/tree/master/charts/incubator/rollarr
   - https://github.com/TheHumanRobot/Rollarr
```

Comment:
```suggestion
```

Reply:
Looks like the docker maintainer has deleted the repo (or made private).
I think we have to yeet this app, unless there is a maintained contaienr

Reply:
> I WILL NOT MAINTAIN THIS REPOSITORY
yea lets yeet this app honestly.